internal/tiltfile/tiltfile_test.go

```diff
@@ -351,10 +364,47 @@ func TestRunTrigger(t *testing.T) {
 		t.Fatal("loading tiltconfig:", err)
 	}
 
-	_, err = tiltconfig.GetServiceConfigs("yarnly")
+	services, err := tiltconfig.GetServiceConfigs("yarnly")
 	if err != nil {
 		t.Fatal("getting service config:", err)
 	}
 
-	// TODO(dmiller): actually test that trigger makes it in to the service definition
+	assert.Equal(t, len(services), 1)
+	assert.Equal(
+		t,
+		services[0].Steps[0].Cmd,
+		model.Cmd{
+			Argv: []string{"sh", "-c", "yarn install"},
+		},
+	)
+	packagePath := td.JoinPath("package.json")
+	matches, err := services[0].Steps[0].Trigger.Matches(packagePath, false)
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.True(
```

Comment:
the formatting here seems weird, is there a reason it can't fit on one line?

Reply:
it totally can be, it was a side effect from when it was much longer. Will fix!